test/behavior/saturating_arithmetic.zig

```diff
@@ -277,11 +275,11 @@ test "saturating multiplication" {
             try testSatMul(i8, -128, -128, 127);
             try testSatMul(i8, maxInt(i8), maxInt(i8), maxInt(i8));
             try testSatMul(i16, maxInt(i16), -1, minInt(i16) + 1);
-            try testSatMul(i128, maxInt(i128), -1, minInt(i128) + 1);
-            try testSatMul(i128, minInt(i128), -1, maxInt(i128));
+            if (!spirv) try testSatMul(i128, maxInt(i128), -1, minInt(i128) + 1);
+            if (!spirv) try testSatMul(i128, minInt(i128), -1, maxInt(i128));
             try testSatMul(u8, 10, 3, 30);
             try testSatMul(u8, 2, 255, 255);
-            try testSatMul(u128, maxInt(u128), maxInt(u128), maxInt(u128));
+            if (!spirv) try testSatMul(u128, maxInt(u128), maxInt(u128), maxInt(u128));
```

Comment:
```zig
    try testSatMul(u8, 10, 3, 30);
    try testSatMul(u8, 2, 255, 255);
    if (builtin.zig_backend != .stage2_spirv) {
        try testSatMul(i128, maxInt(i128), -1, minInt(i128) + 1);
        try testSatMul(i128, minInt(i128), -1, maxInt(i128));
        try testSatMul(u128, maxInt(u128), maxInt(u128), maxInt(u128));
    }
```

Reply:
it can be an early return
```zig
    try testSatMul(u8, 10, 3, 30);
    try testSatMul(u8, 2, 255, 255);

    if (builtin.zig_backend == .stage2_spirv) return error.SkipZigTest
    
    try testSatMul(i128, maxInt(i128), -1, minInt(i128) + 1);
    try testSatMul(i128, minInt(i128), -1, maxInt(i128));
    try testSatMul(u128, maxInt(u128), maxInt(u128), maxInt(u128));
```